edx_solutions_api_integration/users/tests.py

```diff
@@ -477,6 +477,17 @@ def test_user_list_get_multiple_filters(self, store):
         self.assertEqual(len(response.data), 1)
         self.assertEqual(response.data[0]['id'], unicode(course1.id))
 
+        # fetch user data by exact course id match, and filter by list of usernames
+        response = self.do_get('{}?username={}&course={}'.format(
+            test_uri,
+            ','.join([user.username for user in users[0:2]]),
+            unicode(course1.id),
+        ))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.data['results']), 2)
+        self.assertEqual(response.data['results'][0]['full_name'], 'John Doe')
+        self.assertEqual(response.data['results'][1]['full_name'], 'Micheal Mcdonald')
+
```

Comment:
Nit: Consider adding a test to verify truncation of queryset.  (Override the setting from 800 to 3, and make a request that would return four names, verify that only 3 are returned).